libs/vue/src/utils.ts

```diff
@@ -61,3 +61,27 @@ export function checkUnsupportedConfig(
     );
   }
 }
+
+export function resolveConfigureWebpack(
+  vueNxConfigPath: string,
+  projectRoot: string
+) {
+  let vueConfig = {
+    configureWebpack: {},
```

Comment:
The required webpack file should not have a `configureWebpack` property. The export should be an anonymous object or function. For example:

```
module.exports = {
  // webpack mods
};
```

or

```
module.exports = config => {
  // webpack mods
};
```